person.rb

```diff
@@ -26,5 +27,7 @@ def of_age?
 
   def can_use_services?
     @parent_permission || is_of_age?
+
+    false
   end
```

Comment:
- [ ] `add_rentals` method should be present here.
`hint`:

def add_rental(book, date)
...

     end